README.md

```diff
@@ -79,6 +79,10 @@ Configuration
   If `False`, run 0 pods, effectively disabling the deployment. (Default: `True`).
 * `HASTEXO_ENABLE_REAPER`: If `True`, run 1 pod in the `hastexo-xblock-reaper` deployment. 
   If `False`, run 0 pods, effectively disabling the deployment. (Default: `True`).
+* `HASTEXO_ASGI_APPLICATION`: Definition for the asgi application,
+  added for backwards compatibility. For the hastexo-xblock<=6.2.0,
```

Comment:
Make this reference `hastexo-xblock<7`. Since the changelog for https://github.com/hastexo/hastexo-xblock/pull/233 identifies a breaking change, we'll have to bump the major for the new XBlock release.

Reply:
Ack, done.